hardware/TLE8110ED_breakout_board/TLE8110ED.bak

```diff
@@ -0,0 +1,394 @@
+EESchema Schematic File Version 2
```

Comment:
we ususally do not need .bak files - let's remove next time we make changes, or I will just remove locally later